game/outfits.rpy

```diff
@@ -1736,6 +1748,12 @@ init -1 python in jn_outfits:
         unlocked=False,
         is_jn_wearable=True
     ))
+    __register_wearable(JNNecklace(
+        reference_name="jn_necklace_necktie",
+        display_name="Necktie",
```

Comment:
Do we see the potential to introduce more necktie-style items in the future? If so, we might be helping ourselves if we adjusted the name to be a little more descriptive! Since this is meant to be worn as part of a work set, perhaps `formal_necktie` would work better in place of just `necktie`?